dnase/trackhub/update_tracks.bash

```diff
@@ -294,12 +302,19 @@ else
     blatport=0
 fi
 
-if [ ${blatport} -ne 0 ]; then
+if [ "${blatport}" != "0" ]; then
```

Comment:
is this a stylistic or functional change?

Reply:
Just stylistic.